lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -85,6 +85,49 @@
   <hr>
 </div>
 
+% if course.entrance_exam_enabled:
+<div class="entrance-exam-grade-container action-type-container">
+  <h2>${_("Entrance Exam grade adjustment")}</h2>
```

Comment:
I recommend having @cptvitamin look at the accessibility of this. I would expect this to be using labels and not headings, for example.